### ISSUE TITLE
error 0x80030002 %1 could not be found

### ISSUE THREAD

**Issue by Yuki7k** [Dijji/FileMeta#75]
After changing the the .zip extension to use the file meta handler via the simple profile I am unable to add a tag to zip files.

The properties > details tab is changed as expected as is the details pane view.

However whenever I add a tag on either view and attempt to commit it via apply or save buttons I get an error pop up window with the following details,

An unexpected error is keeping you from applying properties to this folder. If you continue to recieve this error, you can use the error code to search for help with this problem.

Error 0x80030002 %1 could not be found.

**Comment by Dijji** [comment]
You are right. Other users have found that some NAS drives have rather patchy NTFS implementations. The problem is missing support for built-in property storages in alternate streams. The Windows API allows such storages to be opened with a single call, but of course it will only work if the underlying firmware knows how to play the game. It sounds like you are on doing exactly the right things to try to find a solution. 

Dijji

**Comment by ni1esh** [comment]
You guys are facing problems in other things, here's what's happening with me :- 
I have been trying to send files to my phone, like transferring it from my laptop to phone's internal storage or SD card. And when I try to delete few stuffs of my phone, from my laptop, it shows "error 0x80030002 %1 not found". 
GAAH! This thing is irritating! 💢 someone please help 😞
